_config.yml

```diff
@@ -1,5 +1,7 @@
 baseurl: "/Handbuch-Jugend-Hackathons"
 
+github_hosted: false
```

Comment:
What is the purpose of the variable?

> Move scripts to the footer for better loading time

Would you agree that the intention is "load_content_before_scripts"?
You could comment on your idea here because this will be lost after the PR is merged.
What do you think?

Reply:
No, this variable is to indicate that you are using GitHub Pages to host this page and not a normal webhost. If you are using GitHub Pages, you can use cool variables like `site.github.build_revision` to always load the newest versions of your scripts and style sheets, but if you are not using GitHub Pages there will be an empty `?v` parameter on imports, so this variable is to prevent this. But I will improve it, so you don't have to set this variable and Jekyll will automatically detect if the `?v` parameter can be set.

Thank you!

Reply:
Note for me: Fixed!

Reply:
I think, this variable can be removed now.